style.css

```diff
@@ -0,0 +1,192 @@
+* {
+  font-family: "Inter", sans-serif;
+}
+
+.hero {
+  height: 100vh;
+  width: 100%;
+  min-height: 500px;
+  background-image: url("../images/background.png");
```

Comment:
- Currently, the background image for this section is not displayed, this is because you are using a wrong URL. Please use just one `.` when navigating into your image folder i.e `./images/background.png`. Please fix this 👍 👍 

Your Project 👇 :
<img width="288" alt="gef" src="https://user-images.githubusercontent.com/73607512/200133425-6b22b079-e5e8-4c00-b0c8-812b18ba5efa.PNG">

Figma Design:
<img width="142" alt="ddvdv" src="https://user-images.githubusercontent.com/73607512/200133477-b1ef7c33-09f9-4aa1-9986-435c6cdc2df0.PNG">